src/main/resources/db/changelog/changes/09-create-book-category-table.yml

```diff
@@ -0,0 +1,28 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-book-category-table
+      author: Olya Starykova
+      changes:
+        - createTable:
+            tableName: books_categories
+            columns:
+              - column:
+                  name: book_id
+                  type: bigint
+                  constraints:
+                    foreignKeyName: fk_books_categories_books
+                    referencedTableName: books
+                    referencedColumnNames: id
+                    nullable: false
+                    unique: true
```

Comment:
Since you are referencing the primary key, you don't need to add `unique: true and  uniqueConstraintName: book_id-category_id_unique`

The same for `foreignKeyName: fk_books_categories_categories`